extensions/sql-database-projects/src/test/project.test.ts

```diff
@@ -389,59 +382,15 @@ describe('Project: sdk style project content operations', function (): void {
 		should(project.folders.length).equal(3);
 
 		// try to exclude a glob included folder
-		//await project.excludeFolder('folder1\\nestedFolder\\');
+		await project.excludeFolder('folder1\\nestedFolder');
 
 		// verify folder and contents are excluded
 		should(project.folders.length).equal(2);
 		should(project.files.length).equal(11);
-		should(project.files.find(f => f.relativePath === 'folder1\\nestedFolder\\')).equal(undefined);
-
-		// verify sqlproj has glob exclude for folder, but not for files
-		const projFileText = (await fs.readFile(projFilePath)).toString();
-		should(projFileText.includes('<Build Remove="folder1\\nestedFolder\\**" />')).equal(true, projFileText);
-		should(projFileText.includes('<Build Remove="folder1\\nestedFolder\\otherFile1.sql" />')).equal(false, projFileText);
+		should(project.folders.find(f => f.relativePath === 'folder1\\nestedFolder')).equal(undefined);
 	});
 
-	// skipped because exclude folder not yet supported
-	it.skip('Should handle excluding explicitly included folders', async function (): Promise<void> {
```

Comment:
this exclude test isn't useful anymore?

Reply:
Whoops; that should still be in there.  It's _not_ useful any more because this sorta case should be tested down in DacFx, but I'll do the test strip-down in a separate PR as cleanup work